README.md

```diff
@@ -1,11 +1,17 @@
 # Script: Translation Vendor Database
 
-This repository contains a Python program that could help a translation agency to keep track of a vendor database. The idea is to make it easier for project managers to keep track of who the preferred and back-up vendors are for a project, in which languages they work, what their contact details and rates are and other information that might be useful for the PMs.
+## The repository
+This repository has been created by Elmo Geeraerts for the final assignment of an introductory class on python.
 
-The code has been developed by Elmo Geeraerts.
+The repository contains:
+- The `README.md` file which describes the repository and illustrates how to use the code as a module and by running the script;
+- The `tutorial.md` and `tutorial.ipynb` files which show how to actually use the code and the different functionalities;
+- Two `xlsx` files which were generated while creating the `tutorial.ipynb` file. Be aware that if you run the code in the tutorial notebook, some data will be duplicated in the excel if you do not change anything. To avoid this you can delete the two excel files from the directory or you can change the data in the tutorial code to fit your needs.;

```

Comment:
You could also add a line of code yourself to make sure the file is deleted when it should be:

```python
import os
os.remove('path/to/the/file.xlsx')
```